src/resources/search-resource.js

```diff
@@ -5,8 +5,8 @@
 
 export default function SearchResource({apiHandler}) {
   return {
-    get({sort, limit, offset, q, expand = null}) {
-      const params = {sort, limit, offset, q, expand};
+    get({sort = null, limit = null, offset = null, q = null}) {
```

Comment:
`expand` is missing here too, do we even use that parameter?

Reply:
We don't need `expand`. This PR fixes that discrepancy.